packages/parser/README.md

```diff
@@ -53,87 +50,73 @@ npm install @readme/openapi-parser
 }
 ```
 
-### Methods
-
-#### `.validate()`
+### `validate()`
 
 Validates the API definition against the [Swagger 2.0](https://github.com/OAI/OpenAPI-Specification/tree/main/schemas/v2.0), [OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/tree/main/schemas/v3.0), or [OpenAPI 3.1](https://github.com/OAI/OpenAPI-Specification/tree/main/schemas/v3.1) specifications.
 
-If the `validate.spec` option is enabled (it is enabled by default), then this the API definition will also be validated against specific areas that aren't covered by the Swagger or OpenAPI schemas, such as duplicate parameters, invalid component schema names, or duplicate `operationId` values.
+In addition to validating the API definition against their respective specification schemas it will also be validated against specific areas that aren't covered by the Swagger or OpenAPI schemas, such as duplicate parameters, invalid component schema names, or duplicate `operationId` values.
 
 If validation fails an error will be thrown with information about what, and where, the error lies within the API defintiion.
 
-Internally this method invokes [`.dereference()`](#dereference) so the returned object, whether its a Swagger or OpenAPI definition, will be fully dereferenced.
+Internally this method invokes [`dereference()`](#dereference) so the returned object, whether it's a Swagger or OpenAPI definition, will be fully dereferenced.
 
 ```ts
+import { validate } from '@readme/openapi-parser';
+
 try {
-  const api = await OpenAPIParser.validate(myAPI);
+  const api = await validate(myAPI);
   console.log('🍭 The API is valid!');
 } catch (err) {
   console.error(err);
 }
 ```
 
-#### `.dereference()`
+### `.dereference()`
 
 Dereferences all `$ref` pointers in the supplied API definition, replacing each reference with its resolved value. This results in an API definition that does not contain _any_ `$ref` pointers. Instead, it's a normal JSON object tree that can easily be crawled and used just like any other object. This is great for programmatic usage, especially when using tools that don't understand JSON references.
 
 ```ts
-const api = await OpenAPIParser.dereference(myAPI);
+import { dereference } from '@readme/openapi-parser';
+
+const api = await dereference(myAPI);
 
 // The `api` object is a normal JSON object so you can access any part of the
 // API definition using object notation.
 console.log(api.definitions.person.properties.firstName); // => { type: "string" }
 ```
 
-#### `.bundle()`
+### `.bundle()`
 
 Bundles all referenced files and URLs into a single API definition that only has _internal_ `$ref` pointers. This lets you split up your definition however you want while you're building it, but later combine all those files together when it's time to package or distribute the API definition to other people. The resulting defintiion size will be small, since it will still contain _internal_ JSON references rather than being fully-dereferenced.
```

Comment:
```suggestion
Bundles all referenced files and URLs into a single API definition that only has _internal_ `$ref` pointers. This lets you split up your definition however you want while you're building it, but later combine all those files together when it's time to package or distribute the API definition to other people. The resulting definition size will be small, since it will still contain _internal_ JSON references rather than being fully-dereferenced.
```